components/org.wso2.carbon.config/src/main/java/org/wso2/carbon/config/reader/ConfigFileReader.java

```diff
@@ -73,6 +74,18 @@ public final String getFileContent() throws ConfigurationException {
         String customConfigContent;
         if (customConfig != null && (!customConfig.trim().isEmpty())) {
             try {
+                // Change relative paths to absolute
+                Path customConfigGivenPath = Paths.get(customConfig);
+                if (!customConfigGivenPath.isAbsolute()) {
+                    if (System.getProperty("currentDirectory") != null) {
+                        Path currentWorkingDirectory = Paths.get(
+                                System.getProperty("currentDirectory")
```

Comment:
currentDirectory can be moved to an constant

Reply:
Fixed in here https://github.com/wso2/carbon-config/pull/81/commits/9fe3d6b7599cfdc6e4172bd61d9529daa4755eb8